lib/smartsheet/http_client.ex

```diff
@@ -69,10 +69,12 @@ defmodule Smartsheet.HttpClient do
   end
 
   @impl ClientBehaviour
-  def delete_rows(sheet_id, row_ids) do
+  def delete_rows(sheet_id, row_ids, ignore_rows_not_found \\ false) do
     row_ids = Enum.join(row_ids, ",")
 
-    delete("/sheets/#{sheet_id}/rows", [], params: [ids: row_ids])
+    delete("/sheets/#{sheet_id}/rows", [],
+      params: [ids: row_ids, ignoreRowsNotFound: ignore_rows_not_found]
```

Comment:
i _think_ if you passed it in here as `ignore_rows_not_found`, it would camelize them before sending them to the API. /shrug either way is fine, just noting

Reply:
oh yeah -- bump the version! we should make a dev day ticket so that CI or something else can auto bump it